__tests__/app.test.js

```diff
@@ -176,10 +176,21 @@ describe("/api/articles", () => {
           expect(articles).toBeSortedBy("created_at", { descending: true });
         });
     });
+    test("Status:200 - Articles has comment count property", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles.length).toBe(12);
+          articles.forEach((article) => {
+            expect(article.hasOwnProperty("comment_count")).toBe(true);
```

Comment:
would be cool to see if the comment_count property is carrying the right data type